### PR TITLE
fix(curriculum): ensure left alignment of output in translations

### LINKED ISSUE #49515: RTL Translation needs expected output lines to appear in a code block (basic JS course)
### Describe the Issue

In the challenge file:
https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/escape-sequences-in-strings.md

This code:
<blockquote>FirstLine<br>    \SecondLine<br>ThirdLine</blockquote>

Currently is displayed in English but starting from the right (aligned to the right) in the Arabic translation.
I believe this is because it was not wrapped in code blocks.

It needs to be fixed as it represents the expected output from the program. (which will always be aligned on the left, not the right)

Screenshots included.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/basic-javascript/escape-sequences-in-strings

### Your code

```

```

### Expected behavior

use code blocks around the lines of expected output

### Screenshots

arabic-8

### System

N/A

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by atnakayrus**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

Closes #49515 

The *blockquote* were changed to **codeblocks** to ensure Right-to-left format

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/escape-sequences-in-strings.md [hunk 1]
```diff
@@ -19,7 +19,11 @@ Quotes are not the only characters that can be <dfn>escaped</dfn> inside a strin
 
 Assign the following three lines of text into the single variable `myStr` using escape sequences.
 
-<blockquote>FirstLine<br>    \SecondLine<br>ThirdLine</blockquote>
+```
```
Comment (gikf) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/escape-sequences-in-strings.md:22]:
```suggestion
<pre>
```

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/escape-sequences-in-strings.md [hunk 2]
```diff
@@ -19,7 +19,11 @@ Quotes are not the only characters that can be <dfn>escaped</dfn> inside a strin
...
+FirstLine
+    \SecondLine
+ThirdLine
+```
```
Comment (gikf) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/escape-sequences-in-strings.md:26]:
```suggestion
</pre>
```

**Review by gikf — CHANGES_REQUESTED** [review]
Triple backticks cannot be used without specifying language. As the text is pseudocode it can be wrapped in the `<pre></pre>`.

---

As a new contributor, we encourage you to read our [contributing guidelines](https://contribute.freecodecamp.org).

We expect our contributors to be aware of the process specific to this project. Following the guidelines religiously earns you the respect of fellow maintainers and saves everyone time.

Some examples of this are:

1. Do not edit files directly through GitHub – while you can, it's not a good idea.
2. Make sure you follow the PR checklist and not just tick things off; otherwise, we won't take you seriously.
3. Use the correct way to link issues in the description of the PR by updating the `XXXXXX`. Do not just add issue numbers everywhere and anywhere you feel like.
4. Keep mentions and review requests to a minimum. We understand you are excited about contributing, and our maintainers will get back to you as soon as they get a chance.
5. Do not work directly off your `main` branch - create a new branch for the changes you are working on.

We appreciate you taking the time to help us, and we hope to see more contributions from you.

Happy Contributing.

**Comment by gikf** [comment]
Hey @atnakayrus

Could you update the PR title to follow convention described in the contributing guidelines?

**Comment by atnakayrus** [comment]
> Hey @atnakayrus
> 
> Could you update the PR title to follow convention described in the contributing guidelines?

Hey! will "Changing *quoteblocks* to *pre* to ensure left allignment in translated languages" be a good title? if not can you please suggest some title?

**Comment by atnakayrus** [comment]
> Hey @atnakayrus
> 
> Could you update the PR title to follow convention described in the contributing guidelines?

Hey! can you please check if the current title fits the guidelines? if not can you please let me know what would be an appropriate title? Thanks in advance

**Review by gikf — APPROVED** [review]
LGTM :+1:

**Review by Sboonny — APPROVED** [review]
#### :sparkles: :tada: **AWESOME!** :tada: :sparkles:

 Hi @atnakayrus, thanks for this pull request and for contributing to the code-base for the first time. We are looking forward to more contributions from you in the future.

 Cheers & happy contributing!

**Pull request merged**